docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -51,12 +51,14 @@ It is important that the latest version of Cocoapods is installed before the dem
 	`sudo gem install cocoapods`
 
 2. To pull vudrmFairPlay.framework in to the demo project or your project with Cocoapods, ensure that you quit Xcode if it is running, then open the Terminal application and navigate to the project directory, for example:
-
- 	`cd /Users/username/Documents/Dev/vudrm-fairplay-demo-ios`
-
-	Then enter:
+`cd /Users/username/Documents/Dev/vudrm-fairplay-demo-ios`
```

Comment:
```suggestion

`cd /Users/username/Documents/Dev/vudrm-fairplay-demo-ios`
```

Just a heads up that without the extra line the cd example ends up being inline code, going from the rest of the layout I assume that isn't the intention?